packages/@icon-magic/generate/src/plugins/svg-to-raster.ts

```diff
@@ -104,14 +114,36 @@ export const svgToRaster: GeneratePlugin = {
             ? params.propCombo.sizes
             : params.propCombo.sizes.height;
       }
-      const res = params.propCombo.resolutions;
+
+      // If resolution is not passed in (the iterant is not set in the config),
+      // the plugin assumes res = 1 and renames the flavor such that the
+      // resolution is at the end of the flavor's name Eg: if flavor.name =
+      // home-filled@12 then the resulting name will be
+      // home-filled-width-height@12 instead of home-filled@12-width-height
+      let res;
+      let assetName;
+      if (!params.propCombo.resolutions) {
+        res = 1;
+        const resolutionFromName = flavor.name.match(/@[0-9|\.]*/);
+        if (resolutionFromName) {
+          // strip the resolution from the falvor's name and append it to the end
+          assetName = `${flavor.name.replace(
+            resolutionFromName[0],
+            ''
+          )}-${w}x${h}${resolutionFromName[0]}`;
+        } else {
+          // appends @1 to the name
+          assetName = `${flavor.name}-${w}x${h}@${res}`;
+        }
+      } else {
+        res = params.propCombo.resolutions;
```

Comment:
is this not an array?

Reply:
Not when it reaches the plugin. It'll be an array in the plugin-manager, but there will only be one combination of properties that is sent down to the plugin at a time